netbox/overlays/ocp-staging/secrets/s3.conf

```diff
@@ -0,0 +1,5 @@
+[global]
+repo2-s3-endpoint=https://rgw-j-vip.int.massopen.cloud
+repo2-s3-bucket=netbox
+repo2-s3-key=DMOY7XCB20FQ5SX3VLOC
+repo2-s3-key-secret=bmL2BlIaLNUB07C3APcMd0oG2ANulNAguywLaPxx
```

Comment:
shouldn't be in the repo.

Reply:
Ugh, I must have done an `add .` at some point.